### PR TITLE
mydumper: Improve clarity of mydumper warning

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

The way the current warning is written, it is a bit ambiguous as to which maintainers have stopped development. Let's be more careful not to throw mydumper developers under a bus and imply their project is end of life :-)

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/3719
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Review by kennytm — APPROVED** [review]

File: mydumper-overview.md [hunk 1]
```diff
@@ -10,11 +10,7 @@ aliases: ['/docs-cn/dev/mydumper-overview/','/docs-cn/dev/reference/tools/mydump
 
 > **警告：**
 >
-> Mydumper 目前已经不再开发新功能，其绝大部分功能已经被 [Dumpling](/dumpling-overview.md) 取代，强烈建议切换到 Dumpling。
-
-[Mydumper](https://github.com/pingcap/mydumper) 是一个 fork 项目，可以用于对 **MySQL** 或者 **TiDB** 进行逻辑备份，并针对 TiDB 的特性进行了优化。
-
-Mydumper 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/download-ecosystem-tools.md)。
+> [Mydumper](https://github.com/pingcap/mydumper) 是 Pingcap 之前维护的一个 fork 项目，可以针对 TiDB 的特性进行优化。 Mydumper 目前已经不再开发新功能，其绝大部分功能已经被 [Dumpling](/dumpling-overview.md) 取代，且使用 Go 进行了重写。强烈建议切换到 Dumpling。
```
Comment (TomShawn) [inline mydumper-overview.md:13]:
```suggestion
> Pingcap 之前维护的 Mydumper 工具 fork 自 [mydumper project](https://github.com/maxbube/mydumper)，针对 TiDB 的特性进行了优化。Mydumper 目前已经不再开发新功能，其绝大部分功能已经被 [Dumpling](/dumpling-overview.md) 取代。Dumpling 工具使用 Go 语言编写，支持更多针对 TiDB 特性的优化。强烈建议切换到 Dumpling。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: mydumper-overview.md (hunk 1 above)
Comment (ireneontheway) [reply mydumper-overview.md]: 收到

File: mydumper-overview.md [hunk 2]
```diff
@@ -10,11 +10,7 @@ aliases: ['/docs-cn/dev/mydumper-overview/','/docs-cn/dev/reference/tools/mydump
 
 > **警告：**
 >
-> Mydumper 目前已经不再开发新功能，其绝大部分功能已经被 [Dumpling](/dumpling-overview.md) 取代，强烈建议切换到 Dumpling。
-
-[Mydumper](https://github.com/pingcap/mydumper) 是一个 fork 项目，可以用于对 **MySQL** 或者 **TiDB** 进行逻辑备份，并针对 TiDB 的特性进行了优化。
-
-Mydumper 包含在 tidb-enterprise-tools 安装包中，可[在此下载](/download-ecosystem-tools.md)。
+> Pingcap 之前维护的 Mydumper 工具 fork 自 [mydumper project](https://github.com/maxbube/mydumper)，针对 TiDB 的特性进行了优化。Mydumper 目前已经不再开发新功能，其绝大部分功能已经被 [Dumpling](/dumpling-overview.md) 取代。Dumpling 工具使用 Go 语言编写，支持更多针对 TiDB 特性的优化。强烈建议切换到 Dumpling。
```
Comment (lilin90) [inline mydumper-overview.md:13]: Pingcap > PingCAP @ireneontheway

Comment (TomShawn) [reply mydumper-overview.md:13]: Fixed in https://github.com/pingcap/docs-cn/pull/4438. PTAL

**Pull request merged**